src/components/Meme.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
You can remove this react import, in the new version of React it's not necessary to import react. You're allowed to only have this: 
```suggestion
import { useEffect, useState } from 'react';
```